CODE_OF_CONDUCT.md

```diff
@@ -1,56 +1,154 @@
+* Our community spans languages, cultures, perspectives (and continents!), and as such people may not understand jokes, sarcasm, and oblique references in the same way that you do. So remember that and be kind to the other members of the community.
+* Sexist, racist, ableist and other prejudicial or exclusionary comments are not welcome in the community.
+
+### Unacceptable Behavior
+Harassment comes in many forms, including but not limited to: offensive verbal or written comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, and sexual images.
+
+As a community that meets in physical public spaces, harassment also includes: stalking, persistent following, intrusive or otherwise unwanted photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention. 
+
+Examples of other unacceptable behaviors by participants include but are not limited to:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
+advances.
+* Threats or violent language directed against another person.
+* Posting sexually explicit or violent material.
+* Sexist, racist, homophobic, transphobic, ableist, or otherwise discriminatory jokes and language.
+
+* Trolling, insulting/derogatory comments, and personal or political attacks particulary those related to gender, sexual orientation, race, religion or disability.
+* Public or private harassment.
 * Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
-  * Other conduct which could reasonably be considered inappropriate in a
+  address, without explicit permission ("doxing")
+* Other conduct which could reasonably be considered inappropriate in a
     professional setting
 
+If you have any lack of clarity about behaviors we include in the definition of "harassment", please read the [Citizen Code of Conduct](http://citizencodeofconduct.org/). In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.
+
+### Roles
+The following are the various roles of our Community Organizers and the people assigned to each role:
+* The <b>Decider</b> has final say on community guidelines and final authority on correct actions and appeals. The top-level decider is [Adam Jacob](adam@habitat.sh), [@adamhjk](https://twitter.com/adamhjk)
+* The <b>Community Manager</b> guides and meets regularly with community advocates, helps enforce corrective actions, hears appeals, is responsible for maintaining a list of incidents, and ensures pertinent information is shared with necessary parties. The Community Manager is [Ian Henry](ian@habitat.sh), [@eeyun___](https://twitter.com/eeyun___).
```

Comment:
Oh yeah thats totally worth noting. Hadn't thought about local copies.

Can you clarify what you mean about sharing of information?